ccd-definition/CaseField/CareSupervision/createOrderFields.json

```diff
@@ -238,5 +238,30 @@
     "Label": "## Check order details",
     "FieldType": "Label",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "epoExclusionRequirementType",
```

Comment:
make sure that these fields are removed from the case data in the about to submit, see `uk.gov.hmcts.reform.fpl.service.GeneratedOrderService#removeOrderProperties`

Reply:
I think we agreed with Colette and Gail that this radio option will be prepopulated based on what user has selected in the open state, but the fields that aren't pulled in from open state > epoExclusionStartDate, epoWhoIsExcluded.. should both be removed

Reply:
I think we agreed with Colette and Gail that this radio option will be prepopulated based on what user has selected in the open state, but the fields that aren't pulled in from open state > epoExclusionStartDate,

Reply:
oh sorry my bad, this one will need to be removed then also :)

Reply:
I will need some info on this , let's catchup tomorrow

Reply:
sorted